Exam_10.js

```diff
@@ -0,0 +1,215 @@
+// Bài 1: in một bảng nhân của một sô bất kì với các số từ 1 đến 10 và hiện kết quả
+const multiplyNumber = (n) => {
+        let result
+        for (let i = 1; i < 11; i++) {
+            result = n * i
+            console.log(`${n} x ${i} = ${result}`)
+        }
+
+    }
+    // Bài 2:Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 In các số chẵn từ 1 đến n Vd n = 6: in ra: 2 4 6
+const checkEvenNumber = (n) => {
+        n = Number(n)
+
+        let result = []
+        if (n >= 1 && n <= 30) {
+            for (let i = 1; i <= n; i++) {
+                if (i % 2 === 0) {
+                    result[result.length] = i
+                }
+            }
+        }
+        console.log('result=' + result)
+    }
+    // Bài 3:Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 Tính tổng từ 1 đến n
+
+// vd nhập vào 6: Tổng = 1 + 2 + 3 + 4 + 5 + 6 = 21
+const getTotal = (n) => {
+        n = Number(n)
+
+        let result = 0
+        if (n >= 1 && n <= 30) {
+            for (let i = 1; i <= n; i++) {
+                result = result + i
+            }
+        } else { console.log('n không hợp lệ ') }
+        console.log('result=' + result)
+    }
+    // Bài 4
+    // Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 In ra n! (giai thừa của n)
+const getfactorial = (n) => {
+        n = Number(n)
+
+        let factorial = 1
+        if (n >= 1 && n <= 30) {
+            for (let i = 1; i <= n; i++) {
+                factorial = factorial * i
+            }
+        } else { console.log('n không hợp lệ ') }
+        console.log('factorial=' + factorial)
```

Comment:
Nếu viết `log` có thể viết thế này cho đỡ rối:
```suggestion
        console.log('factorial:',  factorial)
```

Thay dấu `+` bằng dấu `,`